portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,124 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  width:1000px;
+  color:#777;
+  font-family:Verdana, Geneva, Tahoma, sans-serif;
+  background-color: lightblue;
+  margin-left:auto;
+  margin-right:auto;
 }
 
-#greeting-container {
+#pix {
+    text-align:center;
```

Comment:
nit: Here and throughout this file, let's use 2 spaces/tabs for property indentation, rather than 4 spaces/tabs. Just so things are nice and consistent throughout the stylesheet :-)